cmd/updateIntegrationArtifactConfiguration_test.go

```diff
@@ -54,10 +54,29 @@ func TestRunUpdateIntegrationArtifactConfiguration(t *testing.T) {
 			ParameterValue:         "def",
 		}
 
-		httpClient := httpMockCpis{CPIFunction: "UpdateIntegrationArtifactConfiguration", ResponseBody: ``, TestType: "Negative", Method: "PUT", URL: "https://demo/api/v1/IntegrationDesigntimeArtifacts(Id='flow1',Version='1.0.1')"}
+		httpClient := httpMockCpis{CPIFunction: "UpdateIntegrationArtifactConfiguration", ResponseBody: ``, TestType: "Negative"}
```

Comment:
[curiosity] Why is the method and the url removed?

Reply:
i added additional test which takes care of complete code coverage based on response body.  so this should ensure complete code coverage for the step